modules/InfiniteStories/StoriesList.module.scss

```diff
@@ -7,6 +7,18 @@
 }
 
 .storiesContainer {
+    @include mobile-only {
+        display: grid;
+        grid-template-columns: repeat(1, 1fr);
+        row-gap: $spacing-6;
+    }
+
+    @include tablet-only {
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+        gap: $spacing-6 $grid-gutter-tablet;
+    }
```

Comment:
The `display` property is the same for all three breakpoints, so it makes sense to move it outside to the root of the selector. The `gap` can also be re-used for mobile and tablet, since mobile only has one column.
And it makes no sense to use `repeat` for mobile where we only have one column, it can be just be `grid-template-columns: 1fr`